### PR TITLE
feat: Update artifacthub-pkg.yml automatically from now on

### PR DESCRIPTION
**Pull request by geeksambhu**
## Description

- Consume github-actions@v3.1.0
- Add new make artifacthub-pkg.yml target, make needed changes to metadata.yml, and recreate artifacthub-pkg.yml.
- Bumped version to 0.2.3

Relates to https://github.com/kubewarden/kubewarden-controller/issues/389

## Test
CI jobs should pass

### CODE REVIEW

**Review by flavio — APPROVED** [review]
LGTM

**Comment by flavio** [comment]
@jvanz can you take a look as well please?

File: Makefile [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
 
 policy.wasm: $(SOURCE_FILES) Cargo.*
 	cargo build --target=wasm32-wasi --release
 	cp target/wasm32-wasi/release/*.wasm policy.wasm
 
+artifacthub-pkg.yml: metadata.yml Cargo.toml
+	kwctl scaffold artifacthub \
+	  --metadata-path metadata.yml --version $(VERSION) \
+	  --questions-path questions-ui.yml --output  artifacthub-pkg.yml
+
```
Comment (jvanz) [inline Makefile:13]: Please, add this [warning](https://github.com/kubewarden/rust-policy-template/blob/01173695123c7bbb358f11287c328f996f621317/Makefile#L9-L12) message in your Makefile.

**Review by jvanz — CHANGES_REQUESTED** [review]
LGTM. But I just want to ask you for one more change.

**Pull request merged**